### PR TITLE
Started on Go-client apps on Confluent Plaform and created a Kafka topic with one partition

### PR DESCRIPTION
**Pull request by parsaloi**
- committing practice project for a go-client on Kafka
- committing my progress in exploring the Confluent Platform

### CODE REVIEW

File: practice-code/kafka-go-getting-started/api-key-6THGZGFF5KR5S5SP.txt
Comment (KingMaina) [inline practice-code/kafka-go-getting-started/api-key-6THGZGFF5KR5S5SP.txt:1]: Is keeping the API keys like this safe? If the implementation allows, let's explore ways of using env variables in Github @Parsaloi

**Review by KingMaina — APPROVED** [review]
Check out my review of the API key file. Other than that, great progress

File: practice-code/kafka-go-getting-started/api-key-6THGZGFF5KR5S5SP.txt
Comment (parsaloi) [reply practice-code/kafka-go-getting-started/api-key-6THGZGFF5KR5S5SP.txt:1]: You are highlighting a very important risk!

Comment (parsaloi) [reply practice-code/kafka-go-getting-started/api-key-6THGZGFF5KR5S5SP.txt:1]: For sure, your insights on that is definitely valuable. Noma sana bro!

**Pull request merged**